src/transport_picoquic.cpp

```diff
@@ -544,7 +544,7 @@ TransportContextId
 PicoQuicTransport::start()
 {
     uint64_t current_time = picoquic_current_time();
-    debug_set_stream(stdout); // Enable picoquic debug
+    //debug_set_stream(stdout); // Enable picoquic debug
```

Comment:
Should we add some ifdef around this in case we want it in debug mode? Or is it just noisy?

I'm all for quieting the library though.

Reply:
It wasn't noisy till recently with the latest picoquic master merge.  It's now pretty noisy. We can add a config option to enable or disable picoquic debug output.